samples/nrf9160/modem_shell/CMakeLists.txt

```diff
@@ -29,3 +29,4 @@ add_subdirectory_ifdef(CONFIG_MOSH_GNSS src/gnss)
 add_subdirectory_ifdef(CONFIG_MOSH_SMS src/sms)
 add_subdirectory_ifdef(CONFIG_MOSH_PPP src/ppp)
 add_subdirectory_ifdef(CONFIG_MOSH_FOTA src/fota)
+add_subdirectory_ifdef(CONFIG_MOSH_SIMPLE_REST_CLIENT src/srest)
```

Comment:
Not related to this line at all but I wanted a conversation to track.

README.rst should be updated to include new feature/command.

Reply:
Added a section about new `rest` command. Thus, welcome @peknis to review.